platforms/package.json

```diff
@@ -18,6 +18,7 @@
       "@ethersproject/base64": "^5.6.1",
       "@ethersproject/providers": "^5.6.2",
       "@gitcoin/passport-types": "^1.0.0",
+      "@spruceid/didkit-wasm": "^0.3.0-alpha0",
```

Comment:
Do we need this dependency in platforms?

Reply:
Yes, for the getAddress function in poh

Reply:
Ok, that is clear now, after the standup.